docs/cli/using-cli.md

```diff
@@ -0,0 +1,81 @@
+# Using the CLI
+
+While a task is running, one of the main concerns for scientists and engineers is
+to understand if the simulation is progressing as expected and/or if there are any errors.
+
+To help with this, the CLI provides a `logs` subcommand that allows users to stream
+the logs of a running task in real time. In this way, running a simulation in a
+powerful remote machine will feel exactly like running in your local computer.
+
+Example:
+
+```bash
+$ inductiva logs f0bnqgf4fcr4asgi4e21tcsqa
+Websocket connected
+Opening socket connection to logs of task f0bnqgf4fcr4asgi4e21tcsqa ...
+timestep: 0.0226
+t/T: 12.2
+breaking: 0
+Fi_iter: 4 Final_residual: 3.73e-09  Fi_time: 0.00174
+breaking: 0
+Fi_iter: 4 Final_residual: 3.08e-09  Fi_time: 0.00172
+breaking: 0
+Fi_iter: 5 Final_residual: 1.68e-11  Fi_time: 0.00167
+umax: 0.102
+vmax: 0
+wmax: 0.0941
+dt: 0.0226
+wavegentime: 2.49e-05
+reinitime: 0
+gctime: 0.000539         average gctime: 0.0005
+Xtime: 0.000126  average Xtime: 0.000641
+total time: 7.62451   average time: 0.00878
+timer per step: 0.00885
+------------------------------------
+869
+simtime: 19.6
+timestep: 0.0226
+t/T: 12.2
+breaking: 0
+Fi_iter: 4 Final_residual: 2.68e-09  Fi_time: 0.00187
+breaking: 0
+Fi_iter: 4 Final_residual: 5.84e-09  Fi_time: 0.00187
+breaking: 0
+Fi_iter: 4 Final_residual: 9.24e-09  Fi_time: 0.00145
+umax: 0.101
+vmax: 0
+wmax: 0.0984
+dt: 0.0226
+wavegentime: 2.06e-05
+reinitime: 0
+gctime: 0.000429         average gctime: 0.0005
+Xtime: 0.000979  average Xtime: 0.000641
+total time: 7.63317   average time: 0.00878
+timer per step: 0.00865
+------------------------------------
+870
+simtime: 19.6
+timestep: 0.0226
+t/T: 12.2
+breaking: 0
+Fi_iter: 4 Final_residual: 2.3e-09  Fi_time: 0.00165
+breaking: 0
+Fi_iter: 5 Final_residual: 5.82e-12  Fi_time: 0.00194
+breaking: 0
+Fi_iter: 4 Final_residual: 3.39e-09  Fi_time: 0.00143
+umax: 0.101
+vmax: 0
+wmax: 0.109
+dt: 0.0226
+wavegentime: 2.39e-05
+reinitime: 0
+gctime: 0.00056  average gctime: 0.0005
+Xtime: 0.00019   average Xtime: 0.000641
+total time: 7.64189   average time: 0.00878
+timer per step: 0.00873
+------------------------------------
+```
+
+This combined with the kill method, allows a quick iteration to stabilize the simulation and achieve the desired configuration.
+
+At the end of the simulation, the logs will be hanging with no messages being logged. In this case, the user can stop the logs by pressing `Ctrl+C`.
```

Comment:
long lines

Reply:
i haven't started with the Using CLI yet!